flink-yarn/src/main/java/org/apache/flink/yarn/AbstractYarnClusterDescriptor.java

```diff
@@ -1189,9 +1189,9 @@ private static Path setupSingleLocalResource(
 					@Override
 					public FileVisitResult visitFile(java.nio.file.Path file, BasicFileAttributes attrs)
 						throws IOException {
-
-						if (!(file.getFileName().startsWith("flink-dist") &&
-								file.getFileName().endsWith("jar"))) {
+						String fileName=file.getFileName().toString();
```

Comment:
missing spaces around `=`, please run checkstyle locally before opening a PR.

Reply:
ok

Reply:
ok，Thanks for reminding